### ISSUE TITLE
install failed on adonis 5.8.7

### ISSUE THREAD

**Issue by hendra1** [jotaajunior/adonis-rabbit#6]
on node_modules there is no build folder
and when trying to 

node ace invoke adonis-rabbit

`Error: Template file not found:`

**Comment by DiegoSouza7** [comment]
Mesmo erro aki, alguma solução apareceu?

**Comment by jotaajunior** [comment]
Sorry for the delay!

This is on me, probably forgot to build before publishing (I really need to automate this).

Meanwhile, for installing you all cand do the following steps:

1. Install `adonis-rabbit` at `0.1.0`
```bash
npm install adonis-rabbit@0.1.0 # npm
yarn add adonis-rabbit@0.1.0 # yarn
```

2. Proceed to invoke adonis-rabbit as usual
```bash
node ace invoke adonis-rabbit
```

Sorry, I'll leave this issue open and close it as soon as I release a new fixed version.

**Comment by jotaajunior** [comment]
Fixed in `0.1.2`. I believe updating you version fixes the issue:

```bash
yarn add adonis-rabbit@latest
```
